go-controller/pkg/ovn/egressgw.go

```diff
@@ -3,11 +3,12 @@ package ovn
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
-	utilnet "k8s.io/utils/net"
 	"net"
 	"strings"
 
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
```

Comment:
nit: this should be grouped with "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"

Reply:
My IDE did that :innocent: